inductiva/molecules/scenarios/protein_solvation/README.md

```diff
@@ -0,0 +1,45 @@
+# Protein Solvation Scenario
+
+This scenario models the dynamics of a protein solvated in water inside a cubic box. The system is modelled with the motion equations for each individual atoms according to the prescription of a molecular force field approximation.
+
+To initilaize the scenario, the user can define the following parameters:
+- Protein .pdb file: PDB file containing the structure of the protein to simulate;
```

Comment:
Again, just say PDB file with the protein, since we are dealing with it to just obtain the structure.